### PR TITLE
Add: Button to lawsuit form fragment - Link to search corporation online

### PR DESCRIPTION
**Pull request by cotom972**

### CODE REVIEW

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java [hunk 1]
```diff
@@ -51,7 +54,13 @@ public View onCreateView(
     receivedDate = rootView.findViewById(R.id.EditText_receivedSpamDate);
     confirmForm = rootView.findViewById(R.id.Button_formFragment_next);
     scrollView = rootView.findViewById(R.id.ScrollView_formFragment);
-
+    searchCompany = rootView.findViewById(R.id.button_formFragment_searchCompany);
+    searchCompany.setOnClickListener(
```
Comment (gulilo) [inline projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java]: Let's try to stay consistent. You are creating `confirmForm` button then `searchCompany` button, but setting the click listener to `search` first.
Either move the `confirm` listener up or move the `search` listener down.
I like to deal with each button then move to the next. So finding it, setting it's listener and what ever else, then moving to the next and doing the same.

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java [hunk 2]
```diff
@@ -28,6 +30,7 @@
   EditText receivedDate;
   TextInputEditText userId;
   TextView idValidationError;
+  Button searchCompany;
```
Comment (gulilo) [inline projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java]: `searchCompany` button is only used in the `OnCreateView` function. It doesn't need to be a class field.
Also true to other views, such as `confirmForm`.

File: projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml [hunk 3]
```diff
@@ -271,8 +287,8 @@
                             android:id="@+id/EditText_receivedSpamDate"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
-                            android:gravity="right"
                             android:focusable="false"
+                            android:gravity="right"
```
Comment (gulilo) [inline projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml]: What is this change?

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java (hunk 1 above)
Comment (cotom972) [reply projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java]: Sounds good :+1:

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java (hunk 2 above)
Comment (cotom972) [reply projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java]: :+1:

File: projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml (hunk 3 above)
Comment (cotom972) [reply projects/noloan/app/src/main/res/layout/lawsuit_fragment_form.xml]: No change... I guess just the "line ordering"...

**Comment by cotom972** [comment]
Hey! Thanks for the comments..... :+1: 
Changed it, see if there's something missing.....

\+ Added some future add-on for validating the company's ID as well, small add-on so I added it "on the go" to this PR... Have a look

**Pull request merged**